types/Cargo.toml

```diff
@@ -62,8 +64,17 @@ std = [
     "schemars",
     "thiserror",
     "num/std",
+    "hashing/std",
+]
+no-std = [
+    "base16/alloc",
+    "hex/alloc",
+    "serde/alloc",
+    "serde_json/alloc",
+    "serde_bytes/alloc",
+    "displaydoc",
+    "hashing/alloc"
```

Comment:
@Fraser999 We ended up calling the `no-std` flag for this hashing library `alloc`, since we saw the other libraries are doing that.

Not for this PR, but maybe we follow suite and rename `no-std` -> `alloc` here for the 1.4.0 release?